web/schemas/ID4B.json

```diff
@@ -466,7 +448,7 @@
     },
     {
         "key": "SampleUnitCell",
-        "type": "float (6 dim)",
+        "type": "float64",
```

Comment:
do you need 64-bit precision here? If not, you may simply put `float` instead of `float64`

Reply:
I need 6 decimal precision which float can provide .. but I just wanted to have an additional cushion, so I put fload64. If you think this is not needed.. then I can update it.